docs/docs-content/clusters/public-cloud/azure/azure-disk-encryption.md

```diff
@@ -0,0 +1,353 @@
+  **Owner** role on the resource group or subscription.
+
+- Create an
+  [Azure Key Vault](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-customer-managed-keys-portal#set-up-your-azure-key-vault)
+  with the following necessary configuration:
+
+  - Resource access:
+
+    - :white_check_mark: **Azure Virtual Machines for deployment**
+
+  - Purge protection:
+    - :white_check_mark: **Enable purge protection**
+
+- Create a
+  [Disk Encryption Set](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-customer-managed-keys-portal#set-up-your-disk-encryption-set)
+  with the encryption type as **Encryption at-rest with a customer-managed key**.
+
+- In the Azure Key Vault, assign the following
+  [access policies](https://learn.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal) to
+  the Disk Encryption Set that you want to use:
+
+  - Key Management Operations:
+
+    - :white_check_mark: **Get**
+
+  - Cryptographic Operations:
+
+    - :white_check_mark: **Unwrap Key**
+    - :white_check_mark: **Wrap Key**
+
+  - If you have designated a user-assigned identity to the Disk Encryption Set, add the same access policies to the
+    user-assigned identity in the Azure Key Vault.
+
+## Enable Disk Encryption
+
+### New Cluster Profile
+
+Use the following steps to enable disk encryption on a
+[new cluster profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-cluster-profiles.md).
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Ensure you are in the correct project scope.
+
+3. From the left **Main Menu**, select **Profiles** and click **Add Cluster Profile**.
+
+4. Fill out the **Basic Information** and ensure **Type** is not set to **Add-on**. Click **Next**.
+
+5. In **Cloud Type**, under **Infrastructure provider**, select **Azure IaaS**. Click **Next**.
+
+6. Select your base OS pack and configure it to your liking. Select **Next layer** when complete.
+
+7. Select the **Palette eXtended Kubernetes** pack as your base Kubernetes pack.
+
+8. In **Configure Pack**, select **Values** in pack details, and click the **\</\>** button to show the YAML editor.
+
+9. On the right-hand side, click the **Presets** drop-down menu, and select the **Enable Encryption Using
+   Customer-Managed Key** option.
+
+10. Scroll to the bottom of the YAML editor to view the additional configuration that was added.
+
+    ```yaml
+    cloud:
+      azure:
+        diskEncryptionSetID: ""
+    ```
+
+11. Fill in the `diskEncryptionSetID` with the Resource ID URI of your Disk Encryption Set.
+
+    <!-- prettier-ignore -->
+    <details>
+    <summary> How to find the Resource ID URI of your Disk Encryption Set </summary>
+
+    1. Log in to the [Azure Portal](https://portal.azure.com/).
+
+    2. In the search bar, look for **Disk Encryption Sets**. Click on the service when found.
```

Comment:
Let's use more explicit language so that the user knows to click on the search bar and type out the Disk query.